test/functional/device/accessibility-e2e-specs.js

```diff
@@ -0,0 +1,91 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { PREDICATE_SEARCH } from '../helpers/element';
+import { MOCHA_TIMEOUT, initSession, deleteSession } from '../helpers/session';
+import { SETTINGS_CAPS, amendCapabilities } from '../desired';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('Accessibility', function() {
+  this.timeout(MOCHA_TIMEOUT);
+
+  let driver, caps;
+
+  beforeEach(function() {
+    caps = amendCapabilities(SETTINGS_CAPS, { 'appium:usePrebuiltWDA': true });
+  });
+
+  afterEach(async function() {
+    await driver.terminateApp('com.apple.Preferences');
+
+    // try to get rid of the driver, so if a test fails the rest of the
+    // tests aren't compromised
+    await deleteSession();
+  });
+
+  async function showAccessibilityTab(driver) {
+    let hasGeneralTab = false;
+    try {
+      // iOS 13 has Accessibility outside the General tab
+      await driver
+        .$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeCell' AND name == 'Accessibility'`)
+        .click();
+    } catch (err) {
+      await driver
+        .$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeCell' AND name == 'General'`)
+        .click();
+      await driver
+        .$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeCell' AND name == 'Accessibility'`)
+        .click();
+      hasGeneralTab = true;
+    }
+    return hasGeneralTab;
+  }
+
+  describe('ReduceMotion', function() {
+    async function getReduceMotion(driver) {
+      const motionButtonCalledReduceMotion = await showAccessibilityTab(driver);
+      await driver
+        .$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeCell' AND name == '${motionButtonCalledReduceMotion ? 'Reduce Motion' : 'Motion'}'`)
```

Comment:
this line seems too long to me

Reply:
fixed https://github.com/appium/appium-xcuitest-driver/pull/1464/commits/ae128f54446004e7fb5c67e85d1aa613b8128265